x-pack/scripts/functional_tests.js

```diff
@@ -5,4 +5,11 @@
  */
 
 require('@kbn/plugin-helpers').babelRegister();
-require('../dev-tools/functional_tests').runFunctionTests();
+require('@kbn/test').runTests(
+  [
+    'x-pack/test/functional/config.js',
+    'x-pack/test/api_integration/config.js',
+    'x-pack/test/saml_api_integration/config.js',
```

Comment:
The SAML tests need their own config since they require specific ES/Kibana configuration, correct?

Reply:
Yes, the idea was to move all that SAML-specific logic from the start/run methods and into config instead.